### DISCUSSION TITLE
Carrousel

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1698] [Q&A]
Hi 
I'm making an app using h2o wave and i need to make a page where i need to show a list of items, and the best way to improve the ux is showing that items in a carrousel. Is it possible to make in wave? if so wich component should i use?

best regards

**Comment by mturoci** [comment]
Not currently possible, but would definitely be a useful widget. Can you please create a feature request describing the features you would like to get from it? Thanks!
